recipes/bfg/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "bfg" %}
+{% set version = "1.12.15" %}
+{% set sha256 = "330af214a0fed320c591afc1046b0f31e8a438f290da09672973aeaa6411b09d" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.jar
+  url: http://repo1.maven.org/maven2/com/madgag/{{ name }}/{{ version }}/{{ name }}-{{ version }}.jar
+  sha256: {{ sha256 }}
+  
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  run:
+    - openjdk
+
+test:
+  commands:
+    - bfg
+
+about:
+  home: https://rtyley.github.io/bfg-repo-cleaner/
+  license: GPL-3.0
```

Comment:
Please add `license_file: LICENSE` to pull the license into the package from the source code.